x-pack/test/security_solution_api_integration/test_suites/entity_analytics/risk_engine/trial_license_complete_tier/telemetry_usage.ts

```diff
@@ -51,79 +51,62 @@ export default ({ getService }: FtrProviderContext) => {
       await deleteAllRules(supertest, log);
     });
 
-    describe('Risk engine not enabled', () => {
-      it('should has empty riskEngineMetrics', async () => {
-        await retry.try(async () => {
-          const stats = await getRiskEngineStats(supertest, log);
-          const expected = {};
-          expect(stats).to.eql(expected);
-        });
+    it('should return empty metrics when the risk engine is disabled', async () => {
+      await retry.try(async () => {
+        const stats = await getRiskEngineStats(supertest, log);
+        expect(stats).to.eql({});
       });
     });
 
-    describe('Risk engine enabled', () => {
-      let hostId: string;
-      let userId: string;
+    // https://github.com/elastic/kibana/issues/183246
+    it('@skipInServerlessMKI should return metrics with expected values when risk engine is enabled', async () => {
+      expect(await areRiskScoreIndicesEmpty({ log, es })).to.be(true);
 
-      beforeEach(async () => {
-        hostId = uuidv4();
-        const hostEvent = buildDocument({ host: { name: 'host-1' } }, hostId);
-        await indexListOfDocuments(
-          Array(10)
-            .fill(hostEvent)
-            .map((event, index) => ({
-              ...event,
-              'host.name': `host-${index}`,
-            }))
-        );
+      const hostId = uuidv4();
+      const hostDocs = Array(10)
+        .fill(buildDocument({}, hostId))
+        .map((event, index) => ({
+          ...event,
+          'host.name': `host-${index}`,
+        }));
 
-        userId = uuidv4();
-        const userEvent = buildDocument({ user: { name: 'user-1' } }, userId);
-        await indexListOfDocuments(
-          Array(10)
-            .fill(userEvent)
-            .map((event, index) => ({
-              ...event,
-              'user.name': `user-${index}`,
-            }))
-        );
+      const userId = uuidv4();
+      const userDocs = Array(10)
+        .fill(buildDocument({}, userId))
+        .map((event, index) => ({
+          ...event,
+          'user.name': `user-${index}`,
+        }));
 
-        await createAndSyncRuleAndAlerts({
-          query: `id: ${userId} or id: ${hostId}`,
-          alerts: 20,
-          riskScore: 40,
-        });
+      await indexListOfDocuments([...hostDocs, ...userDocs]);
 
-        await riskEngineRoutes.init();
+      await createAndSyncRuleAndAlerts({
+        query: `id: ${userId} or id: ${hostId}`,
+        alerts: 20,
+        riskScore: 40,
       });
 
-      afterEach(async () => {
-        await cleanRiskEngine({ kibanaServer, es, log });
```

Comment:
Should we at least call these 3 clean methods on the "after" hook? To make sure we clean the for subsequent test suites.

Reply:
Ah good point 🤦‍♂️